.github/workflows/autoAssignABTT.yml

```diff
@@ -0,0 +1,25 @@
+name: Auto Assign ABTT to Project Board
+
+on:
+  issues:
+    types: [opened]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.ABTT_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to ABTT Project
+    steps:
+    - name: "Add triage and area labels"
+      uses: actions-ecosystem/action-add-labels@v1
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        labels: |
+          Area: tfx-cli
+          triage
+    - name: "Assign issues with 'Area: ABTT' label to project board"
```

Comment:
May be something like that: 'Assign newly opened issues to project board' will be better, since it is not filterring by 'Area: ABTT' label here

Reply:
Fixed!